content/en/logs/processing/parsing.md

```diff
@@ -250,8 +251,8 @@ Other examples:
 | key:"/valueStr"              | `%{data::keyvalue(":", "/")}`                         | {"key": "/valueStr"}                  |
 | /key:/valueStr               | `%{data::keyvalue(":", "/")}`                         | {"/key": "/valueStr"}                 |
 | key:={valueStr}              | `%{data::keyvalue(":=", "", "{}")}`                   | {"key": "valueStr"}                   |
-| key1=value1\|key2=value2     | <code>%{data::keyvalue("=", "", "", "&#124;")}</code> | {"key1": "value1", "key2": "value2"}  |
-| key1="value1"\|key2="value2" | <code>%{data::keyvalue("=", "", "", "&#124;")}</code> | {"key1": "value1", "key2": "value2"}  |
+| key1=value1\|key2=value2     | <code>%{data::keyvalue(&quot;=&quot;, &quot;&quot;, &quot;&quot;, &quot;&#124;&quot;)}</code> | {"key1": "value1", "key2": "value2"}  |
+| key1="value1"\|key2="value2" | <code>%{data::keyvalue(&quot;=&quot;, &quot;&quot;, &quot;&quot;, &quot;&#124;&quot;)}</code> | {"key1": "value1", "key2": "value2"}  |
```

Comment:
```suggestion
| key1="value1"\|key2="value2" | `%{data::keyvalue(&quot;=&quot;, &quot;&quot;, &quot;&quot;, &quot;&#124;&quot;)}` | {"key1": "value1", "key2": "value2"}  |
```

Reply:
Actually I'll revert this and keep the <code> as it's the only working work around I found which allows me to put a `|` in a code section within a table 😢 
hack found on https://stackoverflow.com/questions/17319940/how-to-escape-a-pipe-char-in-a-code-statement-in-a-markdown-table